tests/upgrades/test_classparameter.py

```diff
@@ -137,9 +155,10 @@ def test_post_puppet_class_parameter_data_and_type(self, count, _clean_scenario)
         :expectedresults: The puppet class parameters data and type should be
             intact post upgrade
         """
-
+        sat_hostname = getattr(class_pre_upgrade_data, next(iter(class_pre_upgrade_data))).satellite
+        post_upgrade_sat_host = Satellite(sat_hostname)
         data = _valid_sc_parameters_data()[count - 1]
-        sc_param = entities.SmartClassParameters().search(
+        sc_param = post_upgrade_sat_host.api.SmartClassParameters().search(
             query={'search': f'parameter="api_classparameters_scp_00{count}"'}
         )[0]
         assert sc_param.parameter_type == data['sc_type']
```

Comment:
Since we are now using the fresh satellite, we need to also upgrade that individually !!  

This is a big blocking step for these scenarios!

Lets brainstorm about some of the possibilities of doing upgrade individually.

Reply:
Execute JT `satellite-upgrade` using broker against target VM

Reply:
As we discussed in our previous meeting, it's not necessary to take any action here since the satellite is already enabled with a puppet. @lpramuk, please note that `satellite-upgrade` has already been included in the CI process. We can discuss any potential improvements at a later time.